ring/model.go

```diff
@@ -518,6 +518,18 @@ func (d *Desc) getOldestRegisteredTimestamp() int64 {
 	return result
 }
 
+func (d *Desc) instancesWithTokensCount() int {
+	count := 0
+	if d != nil {
```

Comment:
super-nit: `if d == nil` we can return 0, and then reduce indentation in the following code.